src/components/Bridge/BridgeCard/index.tsx

```diff
@@ -350,7 +375,7 @@ const BridgeCard: React.FC<BridgeCardProps> = (props) => {
       {isChainDrawerOpen && (
         <SelectChainDrawer
           isOpen={isChainDrawerOpen}
-          chains={chainList}
+          chains={drawerType === ChainField.FROM ? chainList?.filter((x) => x.evm) : chainList}
```

Comment:
can you add some comments here in code on why we are doing this so it will be more clear in future also

Reply:
Done